packages/flutter/lib/src/widgets/selectable_region.dart

```diff
@@ -293,6 +297,7 @@ class _SelectableRegionState extends State<SelectableRegion> with TextSelectionD
   }
 
   void _updateSelectionStatus() {
+    widget.onSelectionChange?.call(_selectable?.getSelectedContent());
```

Comment:
This method is called when the selection geometry changes, this include scrolling a scrollable with an existing selection. The selected content can be the same in this case.

Reply:
Hey, thanks for the review :). Could you please elaborate on this case with a small example code? I tried putting a text in scrollable but could not replicate this behavior that is, the method was not called while scrolling.

Reply:
try this

```dart
import 'package:flutter/material.dart';

void main() => runApp(const MyApp());

class MyApp extends StatelessWidget {
  const MyApp({super.key});

  static const String _title = 'Flutter Code Sample';

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      title: _title,
      home: SelectionArea(
        child: Scaffold(
          appBar: AppBar(title: const Text(_title)),
          body: Center(
            child: ListView.builder(
              itemBuilder: (BuildContext context, int index) {
                return Text('text $index');
              },
              itemCount: 1000,
            ),
          ),
        ),
      ),
    );
  }
}
```
select something and scroll

Reply:
There is also some concern about calling getSelectedContent in this method since it can be called every frame when scrolling and the getSelectedContent does a tree walk.

Reply:
[Screencast from 08-05-2022 01:53:01 AM.webm](https://user-images.githubusercontent.com/57670338/182945729-51f5c47d-fbf7-4eb7-930d-ff5e12657caf.webm)

I tried your code on chrome but got the same results. I also tried running it on my android phone and got the same result there. Am I missing something? :thinking:

Reply:
> There is also some concern about calling getSelectedContent in this method since it can be called every frame when scrolling and the getSelectedContent does a tree walk.

Hmm that is a concern :thinking: Maybe I should look for some other way to make it work..

Reply:
I put a print statement in the `_updateSelectionStatus` and it does print out stuff on every frame it scrolled. do you maybe debounce it somehow?

Reply:
![Screenshot from 2022-08-05 02-13-16](https://user-images.githubusercontent.com/57670338/182949753-f79d886e-85f5-4daf-a62e-23923e04b6cc.png)
 
I've just changed this and I get the "update" output only when I select text or change the selected text :thinking:

Reply:
ah I see, just scroll back and forth without scroll the selection off screen. because once it is offscreen the method wouldnt be call

Reply:
Same result :( Tried scrolling back and forth a few times, keeping it on the edge and scrolling with different speeds as well. Couldn't replicate it neither on chrome nor on my phone.

Reply:
This is getting more and more mysterious, I can reproduce it. Do you have the latest master branch? when you test it on mobile, does the toolbar move around when you scrolling? because it relies on this method to update its position

Reply:
Yes, I'm running on the latest master just pulled it a while ago. Here is how it looks on my phone, it just gives the update message only when a selection changes.

https://user-images.githubusercontent.com/57670338/182954047-46171e33-e0ee-46ac-a478-df2125e5714a.mp4